apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1590,10 +1604,18 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                             ".requestObjectSignature.heading") }
                     </Heading>
                     <Hint>
-                        { t("console:develop.features.applications.forms.inboundOIDC.sections" +
-                            ".requestObjectSignature.description", {
-                            productName: config.ui.productName
-                        }) }
+                        <Trans
+                            i18nKey={
+                                "console:develop.features.applications.forms.inboundOIDC.sections" +
+                                ".requestObjectSignature.description"
+                            }
+                            tOptions={ { productName: config.ui.productName } }
+                        >
+                            WSO2 Identity Server supports receiving an OIDC authentication request as
```

Comment:
`WSO2 Identity Server` has to be parameterize

Reply:
`<Trans>` needs a default placeholder. `i18n` string is set as follows.

```
{{productName}} supports receiving ...
```

`tOptions={ { productName: config.ui.productName } }` will replace the `productName`.